smtp-crawler/src/main/java/be/dnsbelgium/mercator/smtp/domain/crawler/SmtpVisit.java

```diff
@@ -0,0 +1,57 @@
+package be.dnsbelgium.mercator.smtp.domain.crawler;
+
+import be.dnsbelgium.mercator.smtp.persistence.entities.CrawlStatus;
+import be.dnsbelgium.mercator.smtp.persistence.entities.SmtpHostEntity;
+import be.dnsbelgium.mercator.smtp.persistence.entities.SmtpVisitEntity;
+import lombok.Getter;
+import lombok.Setter;
+import org.slf4j.Logger;
+
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+@Getter
+@Setter
+public class SmtpVisit {
+  private UUID visitId;
+  private String domainName;
+  private ZonedDateTime timestamp = ZonedDateTime.now();
+  private int numConversations = 0;
+  private List<SmtpHost> hosts = new ArrayList<>();
+  private static final Logger logger = getLogger(SmtpVisitEntity.class);
+  private CrawlStatus crawlStatus;
+
+  public void add(SmtpHost host) {
+    //TODO kijken of de setVisit nodig is
+    host.setVisit(this);
+    hosts.add(host);
+    numConversations++;
```

Comment:
`++numConversations`

Reply:
https://richardstartin.github.io/posts/perf-myths-and-continuous-profiling#never-true-prefer-pre-increment-to-post-increment-for-loop-induction-variables

Even though this is a myth, does it actually cause any harm? Of course not, loops optimised under this delusion are compiled the same way. The only negative scenario would be if a misguided team member were to insist on this practice being followed.